src/python/bot/fuzzers/engine.py

```diff
@@ -107,14 +107,16 @@ def reproduce(self, target_path, input_path, arguments, max_time):
     raise NotImplementedError
 
   def minimize_corpus(self, target_path, arguments, output_dir, input_dirs,
-                      max_time):
+                      reproducers_dir, max_time):
```

Comment:
reproducers_dir feels like confusing name, how about just crashes_dir
also, can we switch order of args
like input_dirs, output_dir, crashes_dir ? feels like input should come first ? or output_dir, crashes_dir, input_dirs ?

Reply:
Hmm, "crashes" seems ambiguous to me -- are we referring to crash stacktrace? or the input that causes it? Leaving as is for now, but can rename if you feel strongly. 

Reordering sounds good -- done!